### PR TITLE
Beautify performance metrics docstrings

### PR DESCRIPTION
**Pull request by BlairLee**
# Peer Review Information

Some of the metrics docstrings aren’t properly parsed. Update those improper docstrings.

# Pull Request Check List

- [x] Updated **documentation** for changed code.

### CODE REVIEW

File: datasetinsights/evaluation_metrics/average_precision_2d.py [hunk 1]
```diff
@@ -1,10 +1,7 @@
-r"""Reference.
-
-We implement the average precision metrics for object detection based on this:
+r"""This module provides some common average precision metrics implementations
```
Comment (adason) [inline datasetinsights/evaluation_metrics/average_precision_2d.py]: The rendered docstring looks like:
Screen Shot 2020-09-22 at 10 47 18 AM

I suggest improve the following things:

1. Hide urls and create hyperlinks. This reduces visual noise that does not break the flow of reading
2. This should be very clear what 2D this module offers. 

```
""" Average Precision metrics for 2D object detection

This module provides average precision metics to evaluate 2D object detection models, such as metrics defined in `coco evaluation <url>`_.  The most commonly used metrics are `MeanAveragePrecisionIOU50 <link>` and `MeanAveragePrecision <link>` which provide average precision for all labels considered.

These metrics are implemented based on `A <uri>` and `B <url>`.

metric `AveragePrecision` provide metrics for a given IOU and label. `AveragePrecisionIOU50` is 
  
"""
```

File: datasetinsights/evaluation_metrics/average_precision_2d.py [hunk 2]
```diff
@@ -216,11 +214,11 @@ def compute(self):
...
 
 class MeanAveragePrecisionIOU50(EvaluationMetric):
-    """2D Bounding Box Mean Average Precision metrics at IOU=50%.
+    """2D Bounding Box Mean Average Precision metrics at :math:`IOU=50\%`.
 
-    This implementation would calculate mAP at IOU=50%.
+    This implementation would calculate mAP at :math:`IOU=50\%`.
 
-    .. math:: mAP^{IoU=50} = mean_{label}AP^{label, IoU=50}
+    .. math:: mAP^{IoU=50}=1/N\sum_{label=1}^{N}AP^{IoU=50}
```
Comment (adason) [inline datasetinsights/evaluation_metrics/average_precision_2d.py]: The math should be improved to reflect typical formula convention.
```
.. math:: mAP(\\text{IOU=50})=\\frac{1}{N_{\\text{label}}}\sum_{\\text{label}}AP(\\text{label}, \\text{IOU=50})

where AP is the `AveragePrecision <link>` metrics competed separately for each label.
```

File: datasetinsights/evaluation_metrics/average_precision_2d.py [hunk 3]
```diff
@@ -247,11 +245,12 @@ class MeanAveragePrecisionAverageOverIOU(EvaluationMetric):
 
     This implementation computes Mean Average Precision (mAP) metric,
     which is implemented as the Average Precision average over all
-    labels and IOU = 0.5:0.95:0.05. The max detections per image is
+    labels and :math:`IOU = 0.5:0.95:0.05`. The max detections per image is
     limited to 100.
 
-    .. math:: mAP^{IoU=0.5:0.95:0.05} = mean_{label,IoU}
-    .. math:: AP^{label, IoU=0.5:0.95:0.05}
+    .. math:: mAP^{IoUs} = 1/10\sum_{IoU=0.5}^{0.95}mAP^{IoU}
```
Comment (adason) [inline datasetinsights/evaluation_metrics/average_precision_2d.py]:
```
mAP = \\frac{1}{N_\\text{IOU}N_\\text{label}}\sum_{\\text{label}, \\text{IOU}}AP(\\text{label}, \\text{IOU})
```

File: datasetinsights/evaluation_metrics/average_recall_2d.py [hunk 4]
```diff
@@ -1,8 +1,5 @@
-r"""Reference.
-
-http://cocodataset.org/#detection-eval
-https://arxiv.org/pdf/1502.05082.pdf
-https://github.com/rafaelpadilla/Object-Detection-Metrics/issues/22
+r"""This module provides some common average recall metrics implementations
```
Comment (adason) [inline datasetinsights/evaluation_metrics/average_recall_2d.py]: Same as `average_precision_2d.py`. Can you improve this?

**Review by adason — CHANGES_REQUESTED** [review]

**Comment by adason** [comment]
@BlairLee What's the latest status of this PR? Can we close it?

**Comment by BlairLee** [comment]
> @BlairLee What's the latest status of this PR? Can we close it?

I update the evaluation metrics docstrings. Now it's clear to users. I think we can merge it.

**Review by adason — APPROVED** [review]

**Pull request merged**